src/client/App.js

```diff
@@ -1,21 +1,28 @@
 import React from "react";
-import { BrowserRouter as Router, Route, Switch } from "react-router-dom";
+import { BrowserRouter as Router, Route, Switch, useParams } from "react-router-dom";
+import AddMeal from "./AddMeal";
 import TestComponent from "./components/TestComponent/TestComponent";
+import MainPage from "./MainPage";
+import MealReservationFunction from "./MealReservationFunction"
+
+export default function App() {
 
-function App() {
   return (
     <Router>
       <Route exact path="/">
-        <p>test</p>
+        <MainPage />
+      </Route>
+      <Route path="/share-meal" children={<AddMeal/>}>
       </Route>
-      <Route exact path="/lol">
-        <p>lol</p>
+      <Route path="/meals/:id" children={<MealReservationFunction />} >  
       </Route>
       <Route exact path="/test-component">
         <TestComponent></TestComponent>
       </Route>
```

Comment:
Dont forget to remove this lines in here. I can still access this page
https://negar-meal-sharing.herokuapp.com/test-component